### PR TITLE
Chan.Part.API IT: test transition from member to follower

### PR DESCRIPTION
**Pull request by wlahti**
#### Type of change

- Test update

#### Description

Test that a member transitions to a follower after being removed from the consenters set for a channel and that it continues to pull the latest blocks for the channel.

Also some miscellaneous test cleanup.

#### Related issues

[FAB-18129](https://jira.hyperledger.org/browse/FAB-18129)

### CODE REVIEW

File: integration/raft/channel_participation_test.go [hunk 1]
```diff
@@ -226,35 +226,64 @@ var _ = Describe("ChannelParticipation", func() {
...
 			})
 
-			submitTxn(orderer3, peer, network, orderers, 6, channelparticipation.ChannelInfo{
+			submitTxn(orderer3, peer, network, members, 8, channelparticipation.ChannelInfo{
 				Name:            "participation-trophy",
 				URL:             "/participation/v1/channels/participation-trophy",
 				Status:          "active",
 				ClusterRelation: "member",
-				Height:          7,
+				Height:          9,
```
Comment (tock-ibm) [inline integration/raft/channel_participation_test.go:286]: How about checking the follower (orderer1) again, after this TX?

**Comment by tock-ibm** [comment]
One minor comment, other than that, looks good to me.

File: integration/raft/channel_participation_test.go (hunk 1 above)
Comment (wlahti) [reply integration/raft/channel_participation_test.go:286]: orderer1 is no longer part of the channel at this point. We removed it after we checked that it pulls blocks as a follower.

Comment (tock-ibm) [reply integration/raft/channel_participation_test.go:286]: True... line 266-267... I missed that.

**Comment by tock-ibm** [comment]
Looks good to me.

**Review by tock-ibm — APPROVED** [review]

**Review by C0rWin — APPROVED** [review]

**Pull request merged**